modules/aca/office_booking.rb

```diff
@@ -108,8 +33,6 @@ def on_update
 
         self[:hide_all] = setting(:hide_all) || false
         self[:touch_enabled] = setting(:touch_enabled) || false
-        self[:name] = self[:room_name] = setting(:room_name) || system.name
-
         self[:name] = self[:room_name] = setting(:room_name) || system.name
```

Comment:
`name` and `room_name` do not appear to be used as part of any internal logic so I'm guessing dependencies here only exist in the front end. Is there a need for both?

Reply:
Checking ngx-booking's list of bindings in readme.md (both the current version which is a rewrite, and the latest version prior to the rewrite), the frontend relies on `title` and not `name` or `room_name` for the room name. So I believe name and room_name are currently no longer used for any frontend bindings. Similarly, I also checked meeting_room.rb and skype_logic.rb and can confirm they are not referenced. They are most likely overhangs from very early versions of the driver.

So I will remove name and room_name entirely.

If later we update an existing environment to this version of the driver, and find that it breaks some functionality relating to the system name, then we can simply update that relying component to bind to `title` instead of `name`/`room_name`.